scripts/provision/configure_environment.sh

```diff
@@ -49,4 +49,12 @@ if [[ ${use_nfs_for_synced_folders} -eq 0 ]]; then
     chmod -R 755 /var/www
 fi
 
+status "Installing js build tools"
```

Comment:
Please move this logic to `configure_environment_recurring.sh` to make these tools available in existing installations (not only when installing from scratch). To avoid installation attempts on every vagrant reload, use check similar to https://github.com/paliarush/magento2-vagrant-for-developers/blob/2.0/scripts/provision/configure_environment_recurring.sh#L104